specs/latest/2.0/index.html

```diff
@@ -3443,6 +3443,22 @@ <h3>Preventing undefined behavior with Transform Feedback</h3>
         OpenGL ES 3.0.5 &sect;2.15.2</a>)</span>
     </div>
 
+    <p>
+        In case the same buffer is bound to more than one transform feedback binding point that will
```

Comment:
"active transform feedback binding index, generate INVALID_OPERATION."

Reply:
Used a slightly different wording, but I believe it captures the intent unambiguously.